docs/docs-app/components/app.js

```diff
@@ -1,14 +1,33 @@
-import React from 'react';
+import React, {Component} from 'react';
 
 import Header from './header';
 
 /* eslint-disable react/display-name*/
-export default ({children}) => (
-  <div className="app">
-    <Header />
-    <div className="app-container">
-      {children}
-    </div>
-  </div>
-);
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      opacity: 1,
```

Comment:
Hey. for context, I'm taking a lot of this code from deck.gl / luma.gl / react-map-gl who are going through a redesign. Any of that weird stuff is in the name of compatibility with them going forward.